docs/userguide/execution.rst

```diff
@@ -168,3 +168,49 @@ at 2 tasks :
 at 5 tasks, we overflow as the capacity of a single block is fully used.
 
 .. image:: parsl_parallelism.gif
+
+
+Multi-Site
+----------
+
+.. note::
+   This feature is available since Parsl 0.4.0
```

Comment:
This feature is available since Parsl 0.4.0 -> This feature has been available since Parsl 0.4.0.